### ISSUE TITLE
Reinforced windows are too strong.

### ISSUE THREAD

**Issue by asperger-sind** [space-wizards/space-station-14#10085]
## Description

So, we have reinforced windows which are very common on the station and reinforced walls.
Reinforced windows have 150 hp, reinforced walls have 600, the latter aren't see-through and use 4 plasteel plates and 2 steel.
Reinforced windows on the other hand use 1 steel plate and 2 glass plates.
Now then, this seems to make sense, except for the part where reinforced walls don't have a damage modifier and or it's too weak.

I couldn't find rei wall damage modifiers, I did find rei window however.
Blunt 1.0
Slash 0.3
Piercing 0.6
Heat, Shock 0.0

FLAT REDUCTIONS AS WELL, which is a major issue considering that the strongest weapons in the game deal up to 25~ damage (and that's the high-end, so that, and flat reductions make a lot of weapons in ss14 obsolete and it barely makes sense for a window that's already cheaper, can be shot through via lasers, is completely see-through and it doesn't use sci-fi materials like plasteel, an alloy of steel and plasma (maybe titanium instead of steel? not sure.)

Blunt 5 
Slash 10
Piercing 5

Compare this to the reinforced wall, which has no damage modifiers whatsoever, as it parents from WallBase (which also has no modifiers), and the rei wall itself lacks them as well, effectively putting it on-par with the reinforced window or a bit weaker, whilst the latter can be made as thindows and be stacked essentially in 3 with minimum space loss.

Even if you break a reinforced window, you can fully repair it with the shards that drop, if you break a reinforced wall you get a girder, which gives you 1 steel upon breaking or 2 upon decon, also not very favorable.
**Additional context**

tl;dr
reinforced windows have damage modifiers that make them as strong (or a bit weaker) than rei walls, the latter not having damage modifiers, and from experience generally being weaker even considering that they have 4x the hp.
rei window bad, plsz nerf, too stronk. :tm:

**Comment by Peptide90** [comment]
Reinforced windows have that damage modifier to reduce griefing, they used to be plain paper. This is less an issue now most grief items break when thrown.

 The flat reductions could probably be removed and the destruction behaviour needs modified so you’re less likely be to be able to repair it. If this means increasing our construction materials so be it, we need to stop copying bad design from SS13 if 1 shard = 1 rglass = 1 rwindow, because logically you should always get one shard, so it makes no sense a 4 sided window needing 1 sheet.

**Comment by UbaserB** [comment]
Uhhh wasn't there a structural rework recently, how does this affect the issue?
